packit/config/package_config.py

```diff
@@ -254,6 +255,65 @@ def __eq__(self, other: object):
         )
 
 
+class PackageConfigValidation:
+    def __init__(self, package_config: PackageConfig):
+        from packit.schema import PackageConfigSchema
+
+        self.schema_validation_errors: Dict = {}
+        self.validation_errors: Dict = {}
+
+        config_dict = vars(package_config)
+        del config_dict["_downstream_project_url"]
+        del config_dict["dist_git_clone_path"]
+        for field_name, errors in PackageConfigSchema().validate(config_dict).items():
```

Comment:
I am not sure what fields I should validate. They are already validated in load (plus there is problem with PackageConfigSchema().validate on marshmallow 2, raise ValidationError). I don't think my approach really makes sense.

Is there any difference between _schema error and normal. Right now I am treating them separately.